bundles/AdminBundle/Resources/public/js/pimcore/object/preview.js

```diff
@@ -28,15 +28,27 @@ pimcore.object.preview = Class.create({
 
             this.frameId = 'object_preview_iframe_' + this.object.id;
 
+            let items = [];
+            if(this.object.data.general.previewParams) {
+                let paramPanel = this.getParamsPanel();
+                items.push(paramPanel);
+            }
+
+
+            let iFramePanel = Ext.create('Ext.panel.Panel', {
+                html: '<iframe src="about:blank" style="width: 100%;" onload="' + iframeOnLoad
+                    + '" frameborder="0" id="' + this.frameId + '"></iframe>'
+            });
+            items.push(iFramePanel);
+
             this.layout = Ext.create('Ext.panel.Panel', {
                 title: t('preview'),
                 border: false,
                 autoScroll: true,
                 closable: false,
                 iconCls: "pimcore_material_icon_devices pimcore_material_icon",
                 bodyCls: "pimcore_overflow_scrolling",
-                html: '<iframe src="about:blank" style="width: 100%;" onload="' + iframeOnLoad
-                    + '" frameborder="0" id="' + this.frameId + '"></iframe>'
+                items: items
```

Comment:
please remove extra scroll when PreviewGenerator buttons show up.
![screenshot-divesh pim zone-2020 09 07-12_55_50](https://user-images.githubusercontent.com/5137917/92380506-92e98a80-f109-11ea-9d50-88abb924d7fd.png)

Reply:
any suggestions how to achieve this? :)

Reply:
@ultramegatom please check if this works:
```
let toolbar = [];
if(this.object.data.general.previewParams) {
    let paramPanel = this.getParamsPanel();
    toolbar.push(paramPanel);
}

this.layout = Ext.create('Ext.panel.Panel', {
    title: t('preview'),
    border: false,
    autoScroll: true,
    closable: false,
    iconCls: "pimcore_material_icon_devices pimcore_material_icon",
    bodyCls: "pimcore_overflow_scrolling",
    tbar: toolbar,
    html: '<iframe src="about:blank" style="width: 100%;" onload="' + iframeOnLoad
        + '" frameborder="0" id="' + this.frameId + '"></iframe>'
});
```

Reply:
please check if this works:
```js
let toolbar = [];
if(this.object.data.general.previewParams) {
    toolbar.push(this.getParamsPanel());
}

this.layout = Ext.create('Ext.panel.Panel', {
    title: t('preview'),
    border: false,
    autoScroll: true,
    closable: false,
    iconCls: "pimcore_material_icon_devices pimcore_material_icon",
    bodyCls: "pimcore_overflow_scrolling",
    tbar: toolbar,
    html: '<iframe src="about:blank" style="width: 100%;" onload="' + iframeOnLoad
        + '" frameborder="0" id="' + this.frameId + '"></iframe>'
});
```

Reply:
ping @ultramegatom

Reply:
```suggestion
                tbar: toolbar,
                html: '<iframe src="about:blank" style="width: 100%;" onload="' + iframeOnLoad
                    + '" frameborder="0" id="' + this.frameId + '"></iframe>'
```